### PR TITLE
Set the `1777` permision for `/var/tmp/tmt`

### LINKED ISSUE #537: tmt shared directory owned by first user who runs tmt
Reproducer:
under root run:
\# tmt run -vvvddd provision -h minute -i rhel-8 login finish

Then directory /var/tmp/tmt/ is created with:
rwxr-xr-x. 3 root root 21 Feb  9 04:59 /var/tmp/tmt/
then switch to normal user:
$ tmt run -vvvddd provision -h minute -i rhel-8 login finish
Failed to create workdir '/var/tmp/tmt/run-002' ([Errno 13] Permission denied: '/var/tmp/tmt/run-002')

### PR DESCRIPTION
**Pull request by lukaszachy**
Fix #537

### CODE REVIEW

**Comment by thrix** [comment]
@lukaszachy could we add a test for this please?

**Comment by lukaszachy** [comment]
> @lukaszachy could we add a test for this please?

Very good point, I changed code ;) and added tests

File: tmt/utils.py [hunk 1]
```diff
@@ -662,6 +662,15 @@ def _workdir_init(self, id_: Optional[str] = None) -> None:
             self._workdir_cleanup(workdir)
 
         # Create the workdir
+        # Making sure WORKDIR_ROOT has 1777 permission if recreated
+        if workdir.startswith(WORKDIR_ROOT):
```
Comment (thrix) [inline tmt/utils.py:666]: I would blend these two conditions into one, and we would spare one indentation

**Review by thrix — APPROVED** [review]
otherwise lgtm

File: tmt/utils.py (hunk 1 above)
Comment (lukaszachy) [reply tmt/utils.py]: OKAY, autopep8 collapses it a bit but whatever.

**Comment by lukaszachy** [comment]
@psss:  Ack or Nack ? If you object with this PR I'll close it (no point to leave it open forever)

**Comment by psss** [comment]
Sorry for the delay and thanks for clarification. Sounds good to me. Only one thing: This is the first test which needs `root` permissions so it would always fail when run locally for quick testing. I suggest in 38e7add to enable it in CI and for `full` mode testing. @lukaszachy, is that ok from your point of view?

**Comment by lukaszachy** [comment]
@psss thanks. Changes good look to me

**Review by psss — APPROVED** [review]

**Pull request merged**